sdk/storage/AzureStorageBlob/Source/DataStructures/TransferManager.swift

```diff
@@ -62,16 +62,11 @@ public protocol TransferManager: ResumableOperationQueueDelegate {
 }
 
 public protocol TransferDelegate: AnyObject {
-    func transfer(
-        _ transfer: Transfer,
-        didUpdateWithState state: TransferState,
-        andProgress progress: TransferProgress?
-    )
-    func transfer(_ transfer: Transfer, didUpdateWithState state: TransferState)
-    func transfer(_ transfer: Transfer, didFailWithError error: Error)
-    func transferDidComplete(_ transfer: Transfer)
-
-    func transfers(_ transfers: [Transfer], didUpdateWithState state: TransferState)
+    func transfer(_: Transfer, didUpdateWithState: TransferState, andProgress: TransferProgress?)
+    func transfer(_: Transfer, didUpdateWithState: TransferState)
+    func transfer(_: Transfer, didFailWithError: Error)
+    func transferDidComplete(_: Transfer)
+    func transfers(_: [Transfer], didUpdateWithState: TransferState)
```

Comment:
I'm thinking this one could be removed entirely. I may have already done so on my branch.

Reply:
Yep, I saw that -will be removed as part of the rebase.